src/stack.rs

```diff
@@ -97,6 +110,14 @@ impl Stack {
         Stack(Vec::new())
     }
 
+    pub fn display(&self) {
```

Comment:
Why not implement the debug trait on stack too (with the same content as this function)? This let the caller decide where the output should go e.g., or print a stack as part of a bigger debug string, etc. It seems slightly arbitrary to have a `display()` function instead of implementing `Debug` just for the stack.

Reply:
Indeed, gonna fix that right away